website/docs/docs/collaborate/explore-projects.md

```diff
@@ -162,8 +162,41 @@ Under the the **Models** option, you can filter on model properties (access or m
 
 </Expandable>
 
+<Expandable alt_header="Trust signals for models" lifecycle="preview">
+
+Trust signal icons offer a quick, at-a-glance view of data health when browsing your dbt models in Explorer. These icons indicate whether a model is **Healthy**, **Caution**, **Degraded**, or **Unknown**. For accurate health data, ensure the resource is up-to-date and has had a recent job run.
+
+Each trust signal icon reflects key data health components, such as whether tests were run successfully, whether the resource has a description, whether it has been built in the past 30 days, and more. 
+
+Access trust signals using the search function or when clicking on **Models** under the **Resource** tab. Hover over or click the trust signal to see detailed information.
+
+#### Healthy 
+- Resource is healthy and ran all tests successfully
+- Successfully built in its last run
+- Has a description
+- Has tests configured
+- Has been built in the past 30 days 
+<Lightbox src="/img/docs/collaborate/dbt-explorer/trust-signal-healthy.png" title="Healthy trust signal icon"/>
+
+#### Caution
+- Successfully built in its last run
+- Is missing a description
+- Does not have any tests configured
+- Has been built in the past 30 day
+<Lightbox src="/img/docs/collaborate/dbt-explorer/trust-signal-caution.png" title="Caution trust signal icon"/>
+
+#### Degraded
+- Last run failed or source freshness error
+<Lightbox src="/img/docs/collaborate/dbt-explorer/trust-signals-degraded.jpg" width="50%" title="Degraded trust signal icon"/>
+
+#### Unknown
+- Health information is unknown because the resource hasn't been run recently. 
```

Comment:
What is "recently" in this case? I'm assuming if the resource has never been run before (which we should mention), but in the above, not running in 30 days is a caution, so how long must it be to revert to "unknown"?

Reply:
Roxi addressing